src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -394,4 +394,10 @@
   <data name="InputFileNotSpecified" xml:space="preserve">
     <value>Please specify a nuspec, project.json, project file to use</value>
   </data>
+  <data name="Error_PackFailed" xml:space="preserve">
+    <value>Error packaging '{0}'.</value>
```

Comment:
Is there a more specific message that can be shown? I'd have no idea why packing failed.